### ISSUE TITLE
CLI.pipe issues

### ISSUE THREAD

**Issue by gre** [gre/playCLI#1]
CLI.pipe current implementation seems to lock in some case (command are not closed and nothing is streamed anymore).
1. when chaining multiple CLI.pipe.
2. when triggering a lot of pipe (with the ffmpeg example and opening multiple connections).
3. during some time (~ 2min in the webradio example)

Example for (1):

``` scala
import concurrent.ExecutionContext.Implicits.global
import concurrent.{Future, Await}
import concurrent.duration.Duration
import play.api.libs.iteratee._
import cli.CLI

val enum = Enumerator("test".map(_.toByte).toArray)
val cat = CLI.pipe("cat")
val all = enum &> cat &> cat &> cat |>>> Iteratee.consume[Array[Byte]]()
concurrent.Await.result(all, Duration("2 s")).map(_.toChar).mkString
```

**Comment by gre** [comment]
Something is strange...

The code bellow is actually working on MacOS but not on my Linux!

**Comment by gre** [comment]
The bug (3) is not a bug.

120 seconds is the default timeout of WS ...

https://github.com/playframework/Play20/blob/master/framework/src/play/src/main/scala/play/api/libs/ws/WS.scala#L60

The Bug (1) and (2) seems to only happen on Linux (tested on ArchLinux)..

**Comment by gre** [comment]
Fixed with 005037192d7863575b3bd56eba5116ee53358d71
